ui/dashboardApp/index.js

```diff
@@ -22,7 +22,38 @@ import AppInstanceProfiling from '@lib/apps/InstanceProfiling/index.meta'
 import AppClusterInfo from '@lib/apps/ClusterInfo/index.meta'
 import AppSlowQuery from '@lib/apps/SlowQuery/index.meta'
 
+import { darkmodeEnabled, switchDarkMode } from '@lib/utils/themeSwitch'
+import publicPathPrefix from './publicPathPrefix'
+const path = require('path')
+
+function loadJSON(filelocation, callback) {
+  let xobj = new XMLHttpRequest()
+  xobj.overrideMimeType('application/json')
+  xobj.open('GET', filelocation, true)
+  const p = new Promise((resolve, reject) => {
+    xobj.onreadystatechange = function () {
+      if (xobj.readyState === 4 && xobj.status === 200) {
+        callback(xobj.responseText)
+        resolve()
+      }
+    }
+  })
+  xobj.send(null)
+  return p
+}
+
+async function initManifest() {
+  const location = path.join(publicPathPrefix, 'asset-manifest.json')
```

Comment:
I think you can simply embed the `asset-manifest.json` file in the HTML content. See https://github.com/facebook/create-react-app/blob/f5c3bdb65480f93b2d4a4c2f3214fc50753de434/packages/react-dev-utils/InlineChunkHtmlPlugin.js